### PR TITLE
refact(common): add more construction methods for convenient

### PR DESCRIPTION
**Pull request by chengxin1374**
add some construction methods for more convenient use

### CODE REVIEW

**Comment by VGalaxies** [comment]
> thanks，maybe change the merge branch to master？
> 
> @VGalaxies all need meet？

All the required ctors are now in place 🎈

**Review by imbajin — DISMISSED** [review]
LGTM

File: hugegraph-common/src/main/java/org/apache/hugegraph/event/EventHub.java [hunk 1]
```diff
@@ -49,21 +49,27 @@ public class EventHub {
...
 
     public EventHub(String name) {
         LOG.debug("Create new EventHub: {}", name);
-
         this.name = name;
         this.listeners = new ConcurrentHashMap<>();
         EventHub.init(1);
     }
 
+    public EventHub(String name, int threadSize) {
```
Comment (javeme) [inline hugegraph-common/src/main/java/org/apache/hugegraph/event/EventHub.java:66]: just let `EventHub(String name)` call `EventHub(name, 1)`

File: hugegraph-common/src/main/java/org/apache/hugegraph/rest/RestResult.java [hunk 2]
```diff
@@ -43,6 +43,12 @@ public RestResult(Response response) {
         this.content = response.readEntity(String.class);
     }
 
+    public RestResult(int status, String content, MultivaluedMap<String, Object> headers) {
```
Comment (javeme) [inline hugegraph-common/src/main/java/org/apache/hugegraph/rest/RestResult.java:46]: ditto

**Review by imbajin — APPROVED** [review]

**Review by javeme — APPROVED** [review]

**Pull request merged**